Common/Systems/ModPlayers/WarShieldPlayer.cs

```diff
@@ -23,7 +23,7 @@ public override void PreUpdateMovement()
 			return;
 		}
 
-		WarShield shield = Player.HeldItem.ModItem as WarShield;
+		var shield = Player.HeldItem.ModItem as WarShield;
```

Comment:
this "var" seems kind of random

Reply:
Some stuff may have just been left unnoticed, specially when it comes to formatting. The linter should point out everything though.

Reply:
Was just thinking we dont really use var, at all.
But, true, if it was accepted by the linter it should be fine.